hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -356,6 +365,11 @@ public boolean dropNamespace(Namespace namespace) {
 
   @Override
   public boolean setProperties(Namespace namespace, Map<String, String> properties) {
+    Preconditions.checkArgument(
+        (properties.get(HMS_DB_OWNER_TYPE) == null) == (properties.get(HMS_DB_OWNER) == null),
+        String.format(
```

Comment:
Ah sorry should have been more clear, Preconditions(..., "%s %s", myVarA, myVarB) should work, that's what I meant!  (see the other Iceberg uses for example)

Reply:
Ah, I see.